lib/shopify-cli/context.rb

```diff
@@ -141,15 +141,13 @@ def mkdir_p(path)
       FileUtils.mkdir_p(path)
     end
 
-    # will open a url in a browser if that functionality is available, otherwise
-    # it will simply output to the console a link for the user to either copy/paste
+    # will output to the console a link for the user to either copy/paste
     # or click on.
     #
     # #### Parameters
     # * `uri` - a http URI to open in a browser
     #
     def open_url!(uri)
-      return system("open '#{uri}'") if mac?
       help = <<~OPEN
         Please open {{green:#{uri}}} in your browser
```

Comment:
I know this isn't related to your changes per se, but I think we could improve usability here if we change this text so that the URL is cleanly separated on its own line. Makes it easier to select the text and copy/paste (if your terminal doesn't support click-to-open).

```suggestion
        Please open this link in your browser:
        {{green:#{uri}}}
```